src/common/Constants.cpp

```diff
@@ -766,7 +766,7 @@ const std::string METRIC_ZILLIQA_SCHEMA{
                        "https://opentelemetry.io/schemas/1.2.0")};
 const std::string METRIC_ZILLIQA_SCHEMA_VERSION{ReadConstantString(
```

Comment:
On reflection we should probably move these schemas back into the code, the user has no business editing these, it is just opening us up to problems in the future.